src/components/Error/Error.tsx

```diff
@@ -0,0 +1,59 @@
+import { Button, makeStyles, Typography } from '@material-ui/core';
+import React, { FC } from 'react';
+
+interface ErrorProps {
+  navigateBack: (event: React.MouseEvent<HTMLButtonElement>) => void;
+  variant?: 'Error' | '404';
```

Comment:
I think it would be good to pass in the error. 
```suggestion
  error?: ApolloError;
```
We could determine the error type based off this and possibly show more error information (maybe just in dev).

Similar `ApolloError` handling is done here https://github.com/SeedCompany/cord-field/blob/2191e6167629aa0c49b19272268a475ea66eceb4/src/api/ApolloProvider.tsx#L59-L65

Reply:
passed through the error instance but will leave out the specific error handling for now